internal/plans/retrieve.go

```diff
@@ -38,7 +38,16 @@ func (c *Client) Retrieve() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			plansList, _, err := c.Service.FindPlans(context.Background()).Execute()
+			// type_ := "standard" // string | Filter plans by its plan type (optional)
+			var inc []string
+			var exc []string
+			inc, exc, _, _, _ = c.Servicer.MetalListOptions(nil, nil)
+			request := c.Service.FindPlans(context.Background()).Include(inc).Exclude(exc)
+			filters := c.Servicer.Filters()
+			if filters["type"] != "" {
```

Comment:
The documentation for the FindPlans API request is here: https://github.com/equinix-labs/metal-go/blob/main/docs/PlansApi.md#findplans

The sample code in there shows the 3 filters that are supported by that endpoint according to the spec.